ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/KsqlResource.java

```diff
@@ -347,37 +351,36 @@ public EndpointResponse handleKsqlStatements(
           commandRunnerWarning);
       return EndpointResponse.ok(entities);
     } catch (final KsqlRestException e) {
-      QueryLogger.warn(
+      QueryLogger.info(
```

Comment:
Rolled back to the prior log level, now that I added the info-with-exception variant.